### DISCUSSION TITLE
yolo classify top1 top5

### DISCUSSION THREAD

**Discussion by abhishekmamdapure** [ultralytics/ultralytics#997] [Q&A] [answered]
While training the classification model I am getting **top_5 as 1** all the time, what exactly is the significance of it, are the top_5 rows getting always 100% accuracy, and if that is true what about the top_1 changing everytime, someone please give a clarity of thought.  

<img width="927" alt="Screenshot 2023-02-16 at 09 47 04" src="https://user-images.githubusercontent.com/47754293/219267246-2db3a03b-7be3-4a87-95c4-bc0271ead71a.png">

**Comment by Laughing-q** [comment, marked as answer]
@abhishekmamdapure hi, `top5_acc` means the correct category among the top 5 with the highest prediction probability, and it could be 1 all the time.
BTW `top1_acc` means the correct category among the top 1 with the highest prediction probability.
